Python/Product/Analysis/Values/BuiltinClassInfo.cs

```diff
@@ -81,6 +81,27 @@ public BuiltinClassInfo(IPythonType classObj, PythonAnalyzer projectState)
 
         public string ShortInstanceDescription => InstanceDescription;
 
+        public string FullyQualifiedName {
+            get {
+                if (_type != null) {
+                    if (_type.IsBuiltin) {
+                        return _type.Name;
+                    }
+                    return _type.DeclaringModule.Name + "." + _type.Name;
+                }
+                return null;
+            }
+        }
+
+        public KeyValuePair<string, string> FullyQualifiedNamePair {
```

Comment:
I can't find usage of this property

Reply:
Not yet, but it's the only way to preserve some of this information, and we have consumers of this API that will likely need it.

I can rewrite some of the `FullyQualifiedName` implementation by using this properly if you prefer? :)